packages/portal/document-viewer/src/components/DocumentViewer.cy.ts

```diff
@@ -0,0 +1,24 @@
+// Cypress component test spec file
+
+import { defaultDocument } from '../../fixtures/markdown-data'
+import DocumentViewer from './DocumentViewer.vue'
+
+describe('<DocumentViewer />', () => {
+  it('renders correctly with props', () => {
+    cy.mount(DocumentViewer, {
+      props: {
+        document: defaultDocument as any,
+      },
+    })
+
+    cy.getTestId('document-viewer').should('be.visible')
+    cy.getTestId('default-styles').should('exist')
+  })
+
+  it('renders error state when missing required props', () => {
+    // @ts-ignore - because we are purposely testing the handling of invalid prop values
+    cy.mount(DocumentViewer)
+
+    cy.getTestId('document-viewer-error').should('be.visible')
+  })
+})
```

Comment:
I was asking because this doesn't seem like much test coverage...? Unsure how valuable this test is in its current state.

Reply:
I know the DocumentViewer component doesn't have any other required props or events so I too have been wondering what additional coverage I could add.

Reply:
Could you pass it a document in the tests and verify the document itself renders?

Reply:
For awareness, you won't need line `15` @mptap , that's specifically from the `swagger-ui-web-component`.

Reply:
Gotcha on both suggestions, will implement them now, thanks @adamdehaven !

Reply:
Addressed both @adamdehaven